source/Halibut/Transport/TcpClientManager.cs

```diff
@@ -0,0 +1,61 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Sockets;
+
+namespace Halibut.Transport
+{
+    public class TcpClientManager
```

Comment:
This class doesn't need to be public, and therefore doesn't need to be part of the public API.